plugin-jdbc-snowflake/build.gradle

```diff
@@ -15,6 +15,8 @@ jar {
 dependencies {
     implementation("net.snowflake:snowflake-jdbc:3.23.1")
     implementation project(':plugin-jdbc')
+    implementation("org.bouncycastle:bcprov-jdk18on:1.80")
+    implementation("org.bouncycastle:bcpkix-jdk18on:1.80")
```

Comment:
This one is not on the platform, please add it their and remove the version from here.